### ISSUE TITLE
Incompatible with Intellij IDEA 2023.2

### ISSUE THREAD

**Issue by elliotkhd** [localizely/flutter-intl-intellij#101]

**Comment by francisnnumbi** [comment]
is there any solution yet for this incompatibility issue ?

**Comment by lzoran** [comment]
Thanks for reporting this issue!

We are working on it and plan to roll out a new release that fixes it in the next few days.

While you wait for the new release, there are some workarounds you could try:

1. Revert to a prior version of IntelliJ: Temporarily use a version earlier than 2023.2.

2. Disable the Flutter Intl plugin and manually generate localization files: This can be done by:
    - Disabling the Flutter Intl plugin in the `pubspec.yaml` file:
    ```yaml
    flutter_intl:
      enabled: false
    ```

    - Running the following command to manually generate localization files:
    ```
    flutter pub global run intl_utils:generate
    ```

**Comment by chenjun1127** [comment]
Not compatible with the version of your running IDE (IntelliJ IDEA 2023.2)  

How to solve？

**Comment by lzoran** [comment]
Hi @chenjun1127,

I see there is already a merged PR for this issue in our internal repo. Not sure why it hasn't been released yet. I'll look into speeding things up. Do keep in mind, though, that IntelliJ might take 1-3 days to approve a new plugin version.

**Comment by lzoran** [comment]
This issue should be fixed with Flutter Intl version `1.18.4-2023.2`.

Closing it as resolved. In case of any irregularities, feel free to open it again.
